anime_downloader/util.py

```diff
@@ -68,7 +75,12 @@ def format_search_results(search_results):
     table = [(i+1, v.title, v.pretty_metadata)
              for i, v in enumerate(search_results)]
     table = tabulate(table, headers, tablefmt='psql')
-    table = '\n'.join(table.split('\n')[::-1])
```

Comment:
Needs a default `layout`

Reply:
Gave it a default value of -1 on line 79 now.